fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/sms/service/SmsCampaignDropdownReadPlatformServiceImpl.java

```diff
@@ -81,6 +83,9 @@ public SmsCampaignDropdownReadPlatformServiceImpl(final SmsConfigUtils smsConfig
             if (!responseOne.getStatusCode().equals(HttpStatus.OK)) {
             }
         } catch (Exception e) {
+            //Developer was too lazy to figure out whwat can go wrong?
```

Comment:
you could already change this now? (or later, up to you; later is perfectly fine with me too)